packages/cli/src/cli.ts

```diff
@@ -5,16 +5,20 @@ import { Command } from 'commander';
 import cliSpinners from 'cli-spinners';
 import logUpdate from 'log-update';
 import {
-  writeBaseWebpackProjectFiles,
+  FileOptions,
+  writeWebpackProjectFiles,
   writeOnePageHTMLFile
-} from './templates';
+} from './templates/file-writers';
 
 type CLIOptions = {
   type: 'tsx' | 'jsx' | 'mdx' | 'onepage';
   name: string;
+  lang?: string;
+  port?: number;
 };
 
 let progressInterval: NodeJS.Timer;
+const SPECTACLE_VERSION = '9.2.1';
```

Comment:
I don't like this, but importing the version number from the core library `package.json` introduces other cross complexities. Open to ideas.

Reply:
If there are added complexities, this seems fine. Maybe we can just write a little node-script to automate updating this on release or something.